symbols.xml

```diff
@@ -1220,8 +1224,8 @@
         <global-address name='process_dig' value='0x141026a03'/>
         <global-address name='cursor' value='0x140e60058'/>
         <global-address name='selection_rect' value='0x140e60068'/>
-        <global-address name='gamemode' value='0x140e60088'/>
-        <global-address name='gametype' value='0x140e6008c'/> -4
+        <global-address name='gamemode' value='0x140e6008c'/>
+        <global-address name='gametype' value='0x140e60088'/>
```

Comment:
Yep, I mentioned that on IRC. The interesting thing is, I *think* `sizeof(LARGE_INTEGER)  == sizeof(long long) == sizeof(int64_t)`, so if "long" is always 32 bits on Windows, LARGE_INTEGER[100] should be the same size as long[200].